traffic_monitor/cache/cache.go

```diff
@@ -306,6 +307,17 @@ func (handler Handler) Handle(id string, rdr io.Reader, format string, reqTime t
 		handler.resultChan <- result
 		return
 	}
+	if val, ok := miscStats["plugin.system_stats.timestamp_ms"]; ok {
+		valString := fmt.Sprintf("%s", val)
+		valInt, valErr := strconv.ParseInt(valString, 10, 64)
+		if valErr != nil {
+			log.Errorf("parse error '%v'", valErr)
```

Comment:
Same comment here, just use `log.Error` and string concatenation to avoid reflection for performance reasons.